Libraries/LibGC/Ptr.h

```diff
@@ -238,4 +239,29 @@ struct Traits<GC::Ref<T>> : public DefaultTraits<GC::Ref<T>> {
     }
 };
 
+template<typename T>
+struct Formatter<GC::Ptr<T>> : Formatter<T const*> {
+    ErrorOr<void> format(FormatBuilder& builder, GC::Ptr<T> const& value)
+    {
+        return Formatter<T const*>::format(builder, value.ptr());
```

Comment:
Should this work like the formatter for `Optional<T>` where it prints `*value` if non-null, and otherwise something like "null"?